glide.yaml

```diff
@@ -81,13 +82,16 @@ import:
 - package: github.com/prometheus/procfs
 - package: github.com/cpanato/html2text
 - package: gopkg.in/olivere/elastic.v5
-  version: v5.0.43
+  version: v5.0.45
 - package: github.com/mattermost/gorp
   version: 995ddf2264c4ad45fbaf342f7500e4787ebae84a
 - package: github.com/go-redis/redis
-  version: v6.5.2
+  version: v6.5.7
 - package: github.com/stretchr/testify
-  version: ~1.1.4
```

Comment:
So what's the process for updating these? Are you going through these manually? I'm used to specified a range based on compatibility promises and just using `glide up`. Not really sure how it works when you specify exact versions.